src/components/task-plan/footer.cjsx

```diff
@@ -49,7 +55,9 @@ PlanFooter = React.createClass
     if deleteable
       deleteLink = <BS.Button bsStyle='link' className='-delete' onClick={@onDelete}>Delete</BS.Button>
 
-    if TaskPlanStore.isHomework(id) and not TaskPlanStore.isPublished(id)
+    hasExercises = TaskPlanStore.getExercises(id)?.length
+    if TaskPlanStore.isHomework(id) and not TaskPlanStore.isPublished(id) and not hasExercises
```

Comment:
Why only allow the "Select Problems" button if they have not selected exercises yet? If anything, just change `bsStyle="primary"` to `bsStyle="default"?